src/main/java/nextstep/ladder/domain/result/LadderResult.java

```diff
@@ -0,0 +1,33 @@
+package nextstep.ladder.domain.result;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LadderResult {
+
+    private List<String> results;
```

Comment:
```suggestion
    private final List<String> results;
```
인스턴스 변수는 가능하면  final로 구현할 것을 추천